site/pages/react-native/overview.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Using within React Native applications
+description: A guide on integrating Account Kit within a React Native application
+---
+
+# Using within React Native applications
+
+We've built a simple example using expo [here](https://github.com/alchemyplatform/aa-sdk-rn-expo/tree/main). This guide assumes you're using Expo, but the same principles should apply to a bare React Native app as well.
+
+:::warning
+The Alchemy Signer is currently in Alpha and only supported on Android devices. To get started using the Alchemy Signer in your React Native app, refer to the [React Native Signer](/react-native/signer/overview) documentation. For 3rd party signer support in React Native, refer to your signer provider's documentation.
+:::
```

Comment:
Can we add some copy below linking to the expo and bare RN docs? on the landing page we have these tiles:
<img width="787" alt="image" src="https://github.com/user-attachments/assets/d2b8ecf9-bb2b-4a25-b045-bfb2ff1d3b0a">

wonder if we can add something like that here for "Expo Quickstart" and "React Native Quickstart"